examples/image-repositories/image-repositories.json

```diff
@@ -6,9 +6,16 @@
       "dockerImageRepository": "openshift/ruby-20-centos7",
       "kind": "ImageRepository",
       "metadata": {
-        "name": "ruby-20-centos7"
+        "name": "ruby-20-centos7",
```

Comment:
the changes to this file are no longer valid, @ncdc did you submit a PR with the correct version already? if not we can put those changes here.  @jcantrill  if andy's changes are already merged just remove the changes to this file from your PR.